<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit_4_position"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_3_comp"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_6_address">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_8_code2"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_2_fio"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7_code1">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4_position">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEdit_9_country"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_5_city">
     <property name="text">
      <string>City</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="lineEdit_10_phone1"/>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="lineEdit_7_code1"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEdit_11_phone2"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_ID">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_8_code2">
     <property name="text">
      <string>Second Code</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_9_country">
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3_comp">
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_ID">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2_fio">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_6_address"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_5_city"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10_phone1">
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_11_phone2">
     <property name="text">
      <string>Phone2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
